### PR TITLE
add pvpacademy.net

### PR DESCRIPTION
**Pull request by calledlvkas**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/pvpacademy/manifest.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "server_name": "pvpacademy",
+  "nice_name": "PvPAcademy",
+  "direct_ip": "pvpacademy.net",
+
+  ],
```
Comment (maaaathis) [inline minecraft_servers/pvpacademy/manifest.txt:6]: The manifest has to be a json file. Moreover you cannot close an non existing array here.
```suggestion
```

**Comment by toohard2explain** [comment]
is the server even online?

**Comment by calledlvkas** [comment]
> is the server even online?

yes, the server is online, but it is no longer called vamin.de -> pvpacademy.net

**Comment by toohard2explain** [comment]
> > is the server even online?
> 
> yes, the server is online, but it is no longer called vamin.de -> pvpacademy.net

both not working

**Comment by maaaathis** [comment]
> > is the server even online?
> 
> yes, the server is online, but it is no longer called vamin.de -> pvpacademy.net

But then I don't think vamin.de should be added back as part of this pr, should it?

**Comment by toohard2explain** [comment]
Bild_2023-12-26_213625741
server is also currently in maintenance mode

File: minecraft_servers/pvpacademy/manifest.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "server_name": "pvpacademy",
+  "nice_name": "PvPAcademy",
+  "direct_ip": "pvpacademy.net",
+  "server_wildcards": [
+    "%.pvpacademy.net"
+  ],
+  "social": {
+    "web": "https://pvpacademy.net",
+    "web_support": "https://pvpacademy.net/discord",
```
Comment (maaaathis) [inline minecraft_servers/pvpacademy/manifest.json:10]: "web_support" cannot be a link to a discord. You need a faq page or similar things here. See here: https://github.com/LabyMod/server-media/pull/851#discussion_r1428335162

**Comment by calledlvkas** [comment]
> But then I don't think vamin.de should be added back as part of this pr, should it?

No vamin should not be added again, only pvpacademy.net and this is only in maintenance work until tomorrow so I wanted to add it today

**Comment by maaaathis** [comment]
> > But then I don't think vamin.de should be added back as part of this pr, should it?
> 
> No vamin should not be added again, only pvpacademy.net and this is only in maintenance work until tomorrow so I wanted to add it today

Maybe then you should remove vamin.de out of your pr?

**Comment by calledlvkas** [comment]
> > > But then I don't think vamin.de should be added back as part of this pr, should it?
> > 
> > 
> > No vamin should not be added again, only pvpacademy.net and this is only in maintenance work until tomorrow so I wanted to add it today
> 
> Maybe then you should remove vamin.de out of your pr?

oh sorry, I thought it would be automatically removed when syncing. ^^

File: minecraft_servers/pvpacademy/background@2x.png
Comment (maaaathis) [inline minecraft_servers/pvpacademy/background@2x.png:1]: compress

File: minecraft_servers/pvpacademy/background.png
Comment (maaaathis) [inline minecraft_servers/pvpacademy/background.png:1]: compress

File: minecraft_servers/pvpacademy/background@2x.png
Comment (calledlvkas) [reply minecraft_servers/pvpacademy/background@2x.png:1]: done

File: minecraft_servers/pvpacademy/background.png
Comment (calledlvkas) [reply minecraft_servers/pvpacademy/background.png:1]: done

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**